scripts/steam_age.zs

```diff
@@ -20,16 +32,39 @@ val name_removals = [
     "gtadditions:primitive_assembly"
 ] as string[];
 
+val name_yeets = [
+	<meta_tile_entity:steam_extractor_steel>,
+	<meta_tile_entity:steam_macerator_steel>,
+	<meta_tile_entity:steam_compressor_steel>,
+	<meta_tile_entity:steam_hammer_steel>,
+	<meta_tile_entity:steam_furnace_steel>,
+	<meta_tile_entity:steam_alloy_smelter_steel>,
+] as IItemStack[];
+
 for item in name_removals {
     recipes.removeByRecipeName(item);
 }
+for item in name_yeets {
+    mods.jei.JEI.removeAndHide(item);
+}
 
 recipes.addShaped("gt_coke_clay", <metaitem:compressed.coke.clay> * 8, [[<ore:ingotClay>, <ore:ingotClay>, <ore:ingotClay>],[<ore:sand>, <ore:formWood>, <minecraft:sand>], [<ore:sand>, <ore:sand>, <ore:sand>]]);
-recipes.addShaped("gt_bronze_hull", <gregtech:machine_casing:10>, [[null, <ore:plateBronze>, null],[<ore:plateBronze>, <ore:craftingToolHardHammer>, <ore:plateBronze>], [null, <ore:plateBronze>, null]]);
+recipes.addShaped("gt_coke_hatch", <meta_tile_entity:coke_oven_hatch>, [[null, <ore:plateIron>, null],[<ore:gregHardHammers>, <gregtech:metal_casing:8>, <ore:gregWrenches>],[null, <ore:blockGlassColorless>, null]]);
+recipes.addShaped("gt_bronze_hull", <gregtech:machine_casing:10>, [[null, <ore:plateBronze>, null],[<ore:plateBronze>, <ore:gregHardHammers>, <ore:plateBronze>], [null, <ore:plateBronze>, null]]);
 recipes.addShaped("gt_solar_boiler", <meta_tile_entity:steam_boiler_solar_bronze>, [[null, <ore:blockGlassColorless>, null],[<ore:pipeSmallBronze>, <ore:plateSilver>, <ore:pipeSmallBronze>], [null, <gregtech:machine_casing:11>, null]]);
```

Comment:
Do we maybe want to change the silver to something different here?

Reply:
Now that I think about it, if we gate silver behind the "Deep Dark" we definitively need another metal. No idea which

Reply:
Bronze maybe and change textures

Reply:
I think silver is fine as is. If players *really* want solars, they can buy silver dusts with coins (we can add that).
Worst case I'll do another PR in the future.